ui/src/dashboards/selectors/index.ts

```diff
@@ -59,7 +59,7 @@ export const setTimeToUTC = (date: string): string => {
   if (offset < 0) {
     return moment
       .utc(date)
-      .add(offset, 'minutes')
+      .add(Math.abs(offset), 'minutes')
```

Comment:
```suggestion
      .subtract(offset, 'minutes')
```
seems odd to me to explicitly check for a negative number, then take the absolute value of it :shrug:

Reply:
@hoorayimhelping I hear what you're saying and it's something I considered but wasn't bold enough to try. Let me give it a shot and see if it has the same outcome

Reply:
@Asalem1 here's a method i've used in the past to gain confidence:

1. write a test that triggers the bug
    1. swap out your fix and code from master while writing the test if it helps writing the test
    1. confirm the test fails against master
1. apply your fix and get passing tests reliably
1. add your scary negative subtraction
1. if tests pass, confidence. if tests fail, put it back the way it works 👍 

this works because you go through the steps of triggering the bug against master, then fixing it. that is what gives you the confidence in your solutions *and* your tests.

Reply:
Totally a great idea, unfortunately testing relative time based on a timerange here is not as straightforward as it seems due to the constraints of our current testing framework